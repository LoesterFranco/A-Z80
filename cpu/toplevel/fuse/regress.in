00
0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 00 -1
-1

ed67
3624 b16a a4db b9de 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 ed 67 -1
b9de 93 -1
-1

ed6f
658b 7a7a ecf0 403c 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 ed 6f -1
403c c4 -1
-1

81
f500 0f3b 200d dca6 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 81 -1
dca6 49 -1
-1

cb41
9e00 1b43 954e 7be9 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 cb 41 -1
7be9 f7 -1
-1

cb93
c200 4e05 b3f8 2234 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 cb 93 -1
2234 a0 -1
-1

cbe5
ca00 df0d d588 b48f 0000 0000 0000 0000 0000 0000 0000 0000
00 00 0 0 0 0     1
0000 cb e5 -1
b48f cf -1
-1

