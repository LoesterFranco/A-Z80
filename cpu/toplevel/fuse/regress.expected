00      NOP
    0 MC 0000
    4 MR 0000 00
0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0001
00 01 0 0 0 0 4

ed67    RRD
    0 MC 0000
    4 MR 0000 ed
    4 MC 0001
    8 MR 0001 67
    8 MC b9de
   11 MR b9de 93
   11 MC b9de
   12 MC b9de
   13 MC b9de
   14 MC b9de
   15 MC b9de
   18 MW b9de 69
3324 b16a a4db b9de 0000 0000 0000 0000 0000 0000 0000 0002
00 02 0 0 0 0 18
b9de 69 -1

ed6f    RLD
    0 MC 0000
    4 MR 0000 ed
    4 MC 0001
    8 MR 0001 6f
    8 MC 403c
   11 MR 403c c4
   11 MC 403c
   12 MC 403c
   13 MC 403c
   14 MC 403c
   15 MC 403c
   18 MW 403c 45
6c2d 7a7a ecf0 403c 0000 0000 0000 0000 0000 0000 0000 0002
00 02 0 0 0 0 18
403c 45 -1

81      ADD A,C
    0 MC 0000
    4 MR 0000 81
3031 0f3b 200d dca6 0000 0000 0000 0000 0000 0000 0000 0001
00 01 0 0 0 0 4

cb41    BIT 0,C
    0 MC 0000
    4 MR 0000 cb
    4 MC 0001
    8 MR 0001 41
9e10 1b43 954e 7be9 0000 0000 0000 0000 0000 0000 0000 0002
00 02 0 0 0 0 8

cb93    RES 2,E
    0 MC 0000
    4 MR 0000 cb
    4 MC 0001
    8 MR 0001 93
c200 4e05 b3f8 2234 0000 0000 0000 0000 0000 0000 0000 0002
00 02 0 0 0 0 8

cbe5    SET 4,L
    0 MC 0000
    4 MR 0000 cb
    4 MC 0001
    8 MR 0001 e5
ca00 df0d d588 b49f 0000 0000 0000 0000 0000 0000 0000 0002
00 02 0 0 0 0 8

